src/ProductFilters/WidgetLayeredNav.php

```diff
@@ -34,7 +34,15 @@ protected function layered_nav_list($terms, $taxonomy, $query_type) {
    * @todo Implement output.
    */
   protected function layered_nav_dropdown($terms, $taxonomy, $query_type) {
-    return parent::layered_nav_dropdown($terms, $taxonomy, $query_type);
+    ob_start();
+    $found = parent::layered_nav_dropdown($terms, $taxonomy, $query_type);
```

Comment:
The parent class WC_Widget_Layered_Nav invokes several hooks before and after building the filter links and forms, which should allow you to adjust them without manipulating already rendered HTML.

Reply:
No hooks available. Can do this upstream with woocommerce?